CODE_OF_CONDUCT.md

```diff
@@ -14,22 +14,22 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
```

Comment:
I like this file. Did you get this from somewhere?

Reply:
Yes, this is the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org) which I've taken to adding to any open source projects. It's one of the options GitHub recommends in the Insights -> Community settings section for open source repos.